### PR TITLE
Fix: don’t do a pull, it doesn’t reliably work

### PR DESCRIPTION
**Pull request by pmuir**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/gits/helpers.go [hunk 1]
```diff
@@ -407,13 +407,16 @@ func ForkAndPullPullRepo(gitURL string, baseDir string, baseRef string, branchNa
...
-			if err != nil {
-				return "", "", nil, fork, errors.WithStack(err)
-			}
 		}
-		err = gitter.Pull(dir)
+		err = gitter.Checkout(dir, branchName)
+		if err != nil {
+			return "", "", nil, fork, errors.WithStack(err)
+		}
+		err = gitter.FetchBranch(dir, "orgin", baseRef)
```
Comment (dgozalo) [inline pkg/gits/helpers.go]: Typo on "orgin"

Comment (pmuir) [reply pkg/gits/helpers.go]: Should be fixed.

**Pull request merged**